### DISCUSSION TITLE
Does the CC2652P2 with Ethernet support POE?

### DISCUSSION THREAD

**Discussion by dumaresq** [tube0013/tube_gateways#136] [Q&A] [answered]
I have this device, and I thought it had POE but when I plugged it into a switch that provided POE it didn't work until I added USB power.  I don't know why I thought it had POE, cause I can't find it in the docs now, but can someone confirm if it does or doesn't support POE?

**Comment by tube0013** [comment, marked as answer]
There is a version with and without PoE 

Check the main repo page as it it has links to both with photos of the units. If the micro usb is accessible through an opening in the case which is circular with one flat side where the Ethernet and usb are, that product is not PoE.
